app/views/task.scala.html

```diff
@@ -5,8 +5,8 @@
 @import scala.concurrent.duration._
 @import scala.language.postfixOps
 
-
-@(loggedUser:User, loginInfo:LoginInfo, tasks: Future[Seq[Task]])(implicit request:RequestHeader,messages:Messages)
+@import models.database.TaskDB
```

Comment:
We sould use business models in controllers and views. Thus, I would highly recommend to introduce a `Task` model that is saved as `TaskDB`, if and only if, we're using a SQL based database.

Reply:
this is cleand up by issue #123